spec/system/candidate_interface/candidate_signing_in_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+describe 'A candidate signing in' do
+  context 'who successfully signs in' do
+    before do
+      visit '/'
+      click_on 'sign in'
+
+      fill_in t('authentication.sign_up.email_address.label'), with: 'new_candidate@example.com'
+      # click_on t('authentication.sign_up.button')
+    end
+
+    it 'sees the the sign in page' do
+      expect(page).to have_content 'Enter the email address you used to register with'
+    end
+
+    it 'sees the "Check your email" page' do
+      fill_in t('authentication.sign_in.email_address.label'), with: 'new_candidate@example.com'
+      click_on t('authentication.sign_up.button')
```

Comment:
This looks wrong — could this be `sign_in.button`? The i18n duplication doesn't matter

Reply:
residue of copy&past :)